otj-server-jaxrs/src/main/java/com/opentable/server/RestReactiveHttpServer.java

```diff
@@ -31,5 +31,6 @@
     EmbeddedReactiveJetty.class,
     JAXRSHttpServerCommonConfiguration.class
 })
+@CoreHttpServerCommon
```

Comment:
Also imports EmbeddedJetty (look at CoreHttpServerCommon). I don't think you want to do that.